### PR TITLE
Viite 615 moving between screens

### PR DESCRIPTION
**Pull request by pedrosag**

### CODE REVIEW

File: viite-UI/src/view/RoadAddressProjectForm.js [hunk 1]
```diff
@@ -229,14 +230,102 @@
...
+          currentProject.isDirty = false;
+          jQuery('.modal-overlay').remove();
+          if(!_.isUndefined(result.projectAddresses)) {
+            eventbus.trigger('linkProperties:selectedProject', result.projectAddresses.linkId);
+          }
+          eventbus.trigger('roadAddressProject:openProject', result.project);
+          rootElement.html(selectedProjectLinkTemplate(currentProject, options, selectedProjectLink));
+          _.defer(function(){
+            applicationModel.selectLayer('roadAddressProject');
+            toggleAditionalControls()
```
Comment (plourenco90) [inline viite-UI/src/view/RoadAddressProjectForm.js]: Missing semicolon in ` toggleAditionalControls()`

Comment (plourenco90) [reply viite-UI/src/view/RoadAddressProjectForm.js]: When selecting new link, the navigation button is not visible.

image

**Comment by oappicgi** [comment]
Otherwise seems to work, but I didn't get notification when I  wrote info to new road and clicked on the map to cancel selection.

File: viite-UI/src/view/navigationpanel/ProjectListModel.js [hunk 2]
```diff
@@ -63,6 +63,7 @@
           html += '</table>';
           $('#project-list').html($(html));
           $('[id*="open-project"]').click(function(event) {
+            eventbus.trigger("Opening_road_address_project_" + event.currentTarget.value);
```
Comment (tvenhola) [inline viite-UI/src/view/navigationpanel/ProjectListModel.js]: Trigger name=???

Comment (pedrosag) [reply viite-UI/src/view/navigationpanel/ProjectListModel.js]: That was something I added in order to get some feedback from the application itself when clicking the button (I had to hope it was working to know something did not break) removing in next commit.

**Pull request merged**